src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,7 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User users = new UserService().findByEmail(email);
```

Comment:
userService should be private final field of the class, not a local unnamed variable